hermes-json.cabal

```diff
@@ -60,7 +60,7 @@ library
     Data.Hermes.Decoder.Internal
   build-depends:
     attoparsec         >= 0.13.1 && < 0.15,
-    attoparsec-iso8601 >= 1.0.2.0 && < 1.0.3.0,
+    attoparsec-iso8601 >= 1.0.2.0 && < 1.2,
```

Comment:
Here's the changelog for the new version of attoparsec-iso8601, 1.1.0.0:

https://hackage.haskell.org/package/attoparsec-iso8601-1.1.0.0/changelog

The only thing here is:

> Change parsers of types with year (Day, UTCTime) to require years with at least 4 digits. 

I don't know the hermes-json codebase at all, but I imagine it is possible this would affect something in hermes-json (and possibly require a major version bump in hermes-json??).

Reply:
Yeah it's possible this could break anyone not following the ISO spec? In any case, by way of the new aeson release I found https://hackage.haskell.org/package/text-iso8601, which looks to replace `attoparsec-iso8601`. It might be worthwhile to use that in hermes instead, and then a major bump would be more warranted.